src/megatron/bridge/data/hf_processors/gsm8k.py

```
@@ -0,0 +1,65 @@
+from megatron.bridge.training.tokenizers.tokenizer import MegatronTokenizer
+
+
+def _extract_final_answer(answer: str) -> str:
+    """Extract the final numerical answer after the ``####`` delimiter."""
+    if "####" in answer:
+        return answer.split("####")[-1].strip()
+    return answer.strip()
+
+
+def process_gsm8k_example(
+    example: dict[str, Any], tokenizer: Optional[MegatronTokenizer] = None
```

Ruff (0.15.5)

[warning] Unused function argument: `tokenizer`

(ARG001)